lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -154,6 +176,7 @@ class LighthouseViewerReport {
   onShare() {
     // TODO: find and reuse existing json gist if one exists.
     return this.github.createGist(this.json).then(id => {
```

Comment:
not sure how fast the createGist request is but disable sharebutton before the request so user can get his feedback immediately